.pipelines/templates/template-prod-e2e-steps.yml

```diff
@@ -1,29 +1,55 @@
 parameters:
+  aroVersionStorageAccount: ''
+  azureDevOpsE2EJSONSPN: ''
   location: ''
+  rpMode: ''
   subscription: ''
-  azureDevOpsE2EJSONSPN: ''
 
 steps:
-# TODO(mj) we need to checkout code to run tooling like kubeconfig. This should go away once we refactor everything into golang.
-- template: ./template-checkout.yml
-- template: ./template-az-cli-login.yml
-  parameters:
-    azureDevOpsJSONSPN: ${{ parameters.azureDevOpsE2EJSONSPN }}
 - script: |
     set -e
     export LOCATION=${{ parameters.location }}
     export AZURE_SUBSCRIPTION_ID=${{ parameters.subscription }}
 
-    trap 'rm -f devops-spn.json' EXIT
+    trap 'set +e; rm -f devops-spn.json ; for c in $(docker ps -aq); do docker rm -f $c; done; docker image prune -af; rm -rf ~/.docker/config.json' EXIT
```

Comment:
please make the `rm -rf` a `docker logout -a`, and please add that to .pipelines/templates/template-push-images-to-acr.yml as well

Reply:
OK, ignore me about `docker logout -a` but please add the `rm -rf` to .pipelines/templates/template-push-images-to-acr.yml as well.